coyote/views.py

```diff
@@ -17,13 +17,32 @@
 
 def get_domain(url):
     """
-    Given a url return its domain
+    Parameters:
+    url[string] => uniform resource locator of website
+    --------------------------------------------
+    Returns:
+    parsed_uri.netloc [string] => the domain name in url
     """
+
+    #Given a url return its domain
+
     parsed_uri = urlparse(url)
     return(parsed_uri.netloc)
 
 
 def home(request):
+    """
+    Parameters:
+    request[HttpRequest] 
+    --------------------------------------------
+    Returns:
+    render(request, 'init.html') [HttpResponse] => init.html is returned as HttpResponse
+    """
+
+    #it takes the inputted code from frontend request and sends it to hackerearth API
+    #if the code doesn't compile, then it finds the necessary keyword from error messages 
+    #and searches for it on google with regex matching and suggests debug links 
```

Comment:
Moves these description too inside docstring and I guess we can merge it right away!
Awesome job @142ayushkumar

Reply:
Also, please don't put more than 80 characters in a single line.